python/lsst/afw/fits/SConscript

```diff
@@ -1,3 +1,3 @@
 ## -*- python -*-
 #from lsst.sconsUtils import scripts
-#scripts.BasicSConscript.python(['fitsLib'])
+#scripts.BasicSConscript.pybind11(['fits'])
```

Comment:
In general Pim and I have been leaving the original commented line in all of the Sconscript files and uncommenting and changing them as needed. I think at one point it looked like we would need to wrap the fits.h header but as of now it has not been necessary. However, if we need python to have access to any of those methods this line will need to be uncommented.